src/reuse/modules/util/constants/formatter.constants.ts

```diff
@@ -8,5 +8,17 @@ export enum DateFormats {
   MONTH_DAY_YEAR_COMMA = "mmm dd, yyyy",
   MONTH_DAY_YEAR_COMMA_SHORT = "mmm d, yyyy",
   DATETIME = "datetime",
-  OBJECT = "object"
-}
+  OBJECT = "object",
+  GREGORIAN_DOT = "dd.mm.yyyy (Gregorian Date)",
```

Comment:
Is it necessary to add those brackets with comment in the end, here and in the constants below?

Reply:
before it was, as the response of the odata request is with the brackets, since i cut it now before brackets with replace(), we can probably also delete it from the formatter.constants.ts